### DISCUSSION TITLE
Error in log but sdnext is working ?

### DISCUSSION THREAD

**Discussion by cgidesign-de** [vladmandic/sdnext#2345] [Q&A] [answered]
I get errors when generating images (latest sdnext, windows 11, nvidia rtx 2070):

Multiple lines of the following error are in the log:
| sd | ERROR | sd_vae_approx | Decode simple: Input type (struct c10::Half) and bias type (float) should be the same

Nevertheless images are created.

Any idea what this could mean?

**Comment by vladmandic** [comment]
you're using settings -> live preview -> approximate mode and model format doesn't match it. 
it impacts only live preview.

are you running with --no-half or forced fp32 or something like that?

**Comment by cgidesign-de** [comment]
preview is set to "simple".
medvram is active (using 8GB vram notebook at the moment).

This is what log says about floting point:
Setting Torch parameters: device=cuda dtype=torch.float16 vae=torch.float16 unet=torch.float16
context=no_grad fp16=True bf16=False

**Comment by vladmandic** [comment]
just for my info, what's the gpu? normally autocast should be able to deal with that.
in either case, i think i can patch this tomorrow.

**Comment by cgidesign-de** [comment]
it is an nvidia rtx 2070 notebook - driver is 531.79

**Comment by vladmandic** [comment, marked as answer]
should be fixed now.

**Comment by cgidesign-de** [comment]
yep, error message is gone. Thanks :-)
